newsletter-october-2022.rst

```diff
@@ -0,0 +1,70 @@
+.. post:: Oct 12, 2022
+   :tags: newsletter, python
+   :author: Eric Holscher
+   :location: BND
+
+.. meta::
+   :description lang=en:
+      Company updates and new features from the last month,
+      current focus, and upcoming features.
+
+Read the Docs newsletter - October 2022
+=========================================
+
+September was exciting because a few members of our team finally got to gather in person.
+Manuel, Benjamin, and Eric all attended Djangocon Europe in September,
+and had lots of great discussions around documentation.
+
+Also, as we mentioned in Q4 we're going to be focusing on our core platform features.
```

Comment:
```suggestion
Also, as we mentioned, in Q4 we're going to be focusing on our core platform features.
```

I think a comma here works to separate that "we mentioned this already" and that "this work is going to be done in Q4".